prac_06/languages.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+programs = [ProgrammingLanguage("Python", "Dynamic", True, 1991), ProgrammingLanguage("Ruby", "Dynamic", True, 1995),
```

Comment:
'languages' might be a better name for the list, given that we're creating objects that record data about programming languages specifically (not programs generally)? 

Because you've already created the objects (lines 3-5) you could just use: `programs = [python, ruby, visual_basic]` instead of creating the objects again within your list